crates/biome_yaml_parser/tests/yaml_test_suite/ok/flow/implicit_mapping.yaml

```
@@ -0,0 +1,4 @@
+foo: bar
+"bar": baz
+'baz': qux
+*alias: value
```

YAMLlint (1.38.0)

[error] found undeclared alias "alias"

(anchors)